cms/djangoapps/contentstore/views/preview.py

```diff
@@ -124,6 +125,28 @@ def user_id(self):
         return self._request.user.id
 
 
+class StudioPermissionsService(object):
```

Comment:
I'm not familiar with plans for the "future standard service to make XBlocks aware of users & permissions". As long as platform team is OK with this change, I'm fine with it. Perhaps @cpennington can take a quick look.